README.md

```diff
@@ -34,7 +34,9 @@ You will also need a copy of the [App Engine SDK](https://developers.google.com/
 
 If you want to make changes to the source, you are going to need to run an automated test suite, and for that you will also need [phantomjs](http://phantomjs.org/). Have a look at the testing section for more information.
 
-Note: Java 8 will work locally, and it is also supported by AppEngine, but you may need to [specify your runtime](https://cloud.google.com/appengine/docs/standard/java/runtime-java8) if you want to use appspot.
+Note 1: If you are working on a 64-bit linux systems, you need to install 32-bit versions of: glibc(to get a 32-bit version of ld-linux.so), zlib and libstdc++.
+
+Note 2: Java 8 will work locally, and it is also supported by AppEngine, but you may need to [specify your runtime](https://cloud.google.com/appengine/docs/standard/java/runtime-java8) if you want to use appspot.
```

Comment:
We now specify java8 in the appengine-web.xml configuration, so we should update this note to remove the ", but you may need..." clause.

Reply:
@ewpatton Since this line was about explicitly specifying runtime for using appspot, which is not needed now, should I remove this whole line(instead of just removing "but..." clause)? 
I have not used appspot yet, so still unclear about the process.

Reply:
I would just change appspot to App Engine here. App Engine projects are published under <projectname>.appspot.com.

Reply:
Now that I'm reading it again, we should probably just remove the first sentence of Note 2 entirely. We already specify java 8, and one can't downgrade to Java 7 for App Engine. We still need to keep the parts about certain features such as lambdas not working on Android though.